src/debug_info.rs

```diff
@@ -210,6 +211,59 @@ impl<'ctx> DebugInfoBuilder<'ctx> {
         (builder, cu)
     }
 
+    #[llvm_versions(11.0)]
+    pub(crate) fn new(
+        module: &Module,
+        allow_unresolved: bool,
+        language: DWARFSourceLanguage,
+        filename: &str,
+        directory: &str,
+        producer: &str,
+        is_optimized: bool,
+        flags: &str,
+        runtime_ver: libc::c_uint,
+        split_name: &str,
+        kind: DWARFEmissionKind,
+        dwo_id: libc::c_uint,
+        split_debug_inlining: bool,
+        debug_info_for_profiling: bool,
+        sysroot: &str,
+        sdk: &str,
```

Comment:
Rather than duplicating these functions we should just add `llvm_version` to the params

Reply:
I had to use #[cfg(feature(..)] since custom attributes don't work on parameters.
I also still have to duplicate the function call since attributes on parameter calls are also experimental